### PR TITLE
sakshigrover,dsc umit

### PR DESCRIPTION
**Pull request by sakshigrover-1998**
Fixes #

## Proposed Changes

-
-
-

### CODE REVIEW

**Review by abhinavsri360 — CHANGES_REQUESTED** [review]
Hey @sakshigrover-1998 
Seems like you have entered the wrong geolocations, they point in an ocean currently.
Moreover, please do not include `package-lock.json` in the Pull Request.

**Comment by sakshigrover-1998** [comment]
I did check on the website you send across, it pointed at the right
location. Let me check it again. I'll remove the lock file as well.

Thanks,
Sakshi

**Comment by abhinavsri360** [comment]
@sakshigrover-1998 
Check the json object you sent over, longitude comes first in that 😓

File: src/components/json/worldData.js [hunk 1]
```diff
@@ -488,7 +489,23 @@ const data = [
...
-	}	
+	},
+  /*Sakshi Grover, DSC UMIT, 2020*/
+  {
+	"longitude": 19.0865,
+	"latitude": 72.8305,
+	"batch": 2020,
+	"photo": sakshigrover,
+	"region": "Asia",
+	"skills": ["App Dev", "Web Dev", "Machine Learning", "Cloud", "UI/UX"],
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:500]:
```suggestion
	"skills": ["App Dev", "Web Dev", "AI/ML", "Cloud", "UI/UX"],
```

File: src/components/json/worldData.js [hunk 2]
```diff
@@ -488,7 +489,23 @@ const data = [
 		"linkedin": "https://linkedin.com/in/rajashwin",
 		"twitter": "https://twitter.com/ThisIsAshwinRaj",
 		"portfolio": "http://ashwinraj-in.github.io/"
-	}	
+	},
+  /*Sakshi Grover, DSC UMIT, 2020*/
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:493]:
```suggestion
```

File: src/components/json/worldData.js [hunk 3]
```diff
@@ -488,7 +489,23 @@ const data = [
...
+  {
+	"longitude": 19.0865,
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:495]:
```suggestion
	"longitude": 72.8305,
```

File: src/components/json/worldData.js [hunk 4]
```diff
@@ -488,7 +489,23 @@ const data = [
...
+	"latitude": 72.8305,
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:496]:
```suggestion
	"latitude": 19.0865,
```

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@sakshigrover-1998 I've pointed out the corrections in your PR. Here's what you have to do next:

1. Option 1
    - You can accept and commit the suggestions.
    - Pull the changes to your system.
    - Edit your previous commit and remove your changes in `package-lock.json`.
    - Add and commit changes.
    - Force push and this PR will be overwritten in the right manner.
2. Option 2
    - Close this PR.
    - Make a new PR with the correct changes.

**Pull request closed without merging**